target/linux/ath79/image/generic.mk

```diff
@@ -50,6 +50,30 @@ define Build/cybertan-trx
 	-rm $@-empty.bin
 endef
 
+define Build/belkin_f9j1108-v2-factory
+       cat $@.uImage > $@.new
+       cat $@.rootfs >> $@.new
+       mv $@.new $@
+       rm -rf $@.*
+endef
```

Comment:
I haven't read the discussion on this, but I don't really see why this is separate from edimax-headers.

Since this is doing nothing apart from glueing together what's been created with the other function, edimax-headers won't be much help as a standalone anyway ...

Reply:
I suppose they can be combined again, if you don't think there would be a board where the headers are needed but way the image is put together is unique.

Reply:
> if you don't think there would be a board where the headers are needed but way the image is put together is unique.

If such a thing would come up, we could still split again, and then we would actually know how to split correctly; now, it's just arbitrary.

BTW, I'd also replace the "rm -rf $@.*" by something using the names explicitly.